### PR TITLE
Add UNI token claim detection

### PR DESCRIPTION
**Pull request by FrederikBolding**

### CODE REVIEW

**Comment by FrederikBolding** [comment]
[ch6974]

**Review by blurpesec — COMMENTED** [review]
There doesn't seem to be anything telling you which accounts your UNI tokens are claimable from? Or am i just missing that?

File: src/features/Dashboard/helpers.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
-import { Asset } from '@types';
+import { State as StoreContextState } from '@services/Store/StoreProvider';
 
 import { Action } from './types';
 
-export const filterDashboardActions = (actions: Action[], assets: Asset[]) =>
+export const filterDashboardActions = (actions: Action[], state: StoreContextState) =>
```
Comment (blurpesec) [inline src/features/Dashboard/helpers.ts:5]: Test

File: src/features/Dashboard/constants.ts [hunk 2]
```diff
@@ -3,11 +3,14 @@ import receiveIcon from '@assets/images/icn-receive.svg';
 import sendIcon from '@assets/images/icn-send.svg';
 import swapIcon from '@assets/images/icn-swap.svg';
 import repIcon from '@assets/images/rep-logo.svg';
+import uniIcon from '@assets/images/uni-logo.png';
 import ledgerIcon from '@assets/images/wallets/ledger.svg';
 import trezorIcon from '@assets/images/wallets/trezor.svg';
 import { EXT_URLS, REPV1UUID, ROUTE_PATHS } from '@config';
+import { UNISWAP_LINK } from '@config/links';
```
Comment (blurpesec) [inline src/features/Dashboard/constants.ts:10]: import from `@config`

**Comment by FrederikBolding** [comment]
> There doesn't seem to be anything telling you which accounts your UNI tokens are claimable from? Or am i just missing that?

It does not atm no. I am also not sure where we would show that.

**Comment by blurpesec** [comment]
> It does not atm no. I am also not sure where we would show that.

How about a table like the one we use for ENS?

**Comment by FrederikBolding** [comment]
> > It does not atm no. I am also not sure where we would show that.
> 
> How about a table like the one we use for ENS?

Afaik we wanted to have it as a card on the dashboard instead.

**Review by blurpesec — APPROVED** [review]

**Comment by blurpesec** [comment]
I'm not able to test these because I don't have any UNI claims, but i'm assuming it's working ;)

**Pull request merged**